install/elabftw.sql

```diff
@@ -346,7 +346,10 @@ INSERT INTO `config` (`conf_name`, `conf_value`) VALUES
 ('debug', '0'),
 ('lang', 'en_GB'),
 ('login_tries', '3'),
+('mail_from', ''),
+('mail_method', 'sendmail'),
```

Comment:
Method should not be called sendmail, but rather "default" or "local" or something like that.